### ISSUE TITLE
Proper propagation of operation_id to orchestrations and activity functions

### ISSUE THREAD

**Issue by MarekLani** [Azure/azure-functions-durable-extension#2457]
### Description
When utilizing durable functions and building distributed tracing views with Application Insights I have noticed, oepration_id is not propagated to orchestrations and activities. I observed this behavior with .C# as well as Powershell. I observed, traces captured within trigger functions (which subsequently starts the orchestration) has different operation_id/operation_parent_id than traces captured inside the orchestration. Is this known limitation to be addressed?

**Comment by cgillum** [comment]
@bachuv will this be fixed as part of the distributed tracing work you're finishing up?

**Comment by bachuv** [comment]
@cgillum yes this will be fixed with the distributed tracing work I'm wrapping up.

**Comment by lilyjma** [comment]
@bachuv - has this been fixed in the DTv2 preview?
